app/controllers/users_controller.rb

```diff
@@ -2,4 +2,23 @@ class UsersController < ApplicationController
   def show
     @user = User.find(params[:id])
   end
-end
+
+  def new
+  end
+
+  def create
+    user = User.create(user_params)
+    if user.save
+      redirect_to user_path(user)
+    else
+      redirect_to "/register"
+      flash[:alert] = "Please fill in all fields. Email must be unique."
+    end
+  end
+
+  private
+
+  def user_params
+    params.permit(:name, :email)
+  end
+end
```

Comment:
```suggestion
end

```

Final newline per Rubocop